src/qt/editor/CMakeLists.txt

```diff
@@ -158,33 +170,27 @@ if(IVW_CFG_BUILD_CHANGELOG)
 
     set(srcFile ${IVW_ROOT_DIR}/CHANGELOG.md)
     set(buildFile ${CMAKE_CURRENT_BINARY_DIR}/changelog.html)
-    set(destDir ${IVW_RESOURCES_DIR}/)
-    file(GENERATE 
-        OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/CopyIfFound.cmake"
-        CONTENT 
-            "if (EXISTS \"${buildFile}\")
-                file(COPY \"${buildFile}\" DESTINATION \"${destDir}\")
-            endif()"
-    )
+    set(destFile ${IVW_RESOURCES_DIR}/changelog.html)
 
-    add_custom_command(OUTPUT ${buildFile}
+    add_custom_command(
+        OUTPUT ${buildFile}
         COMMAND ${Python3_EXECUTABLE}
              ${IVW_ROOT_DIR}/tools/changelog-to-html.py 
             -i ${srcFile}
             -o ${buildFile}
+            -o ${destFile}
```

Comment:
same arg `-o`?

Reply:
Ah, multiple -o params are appended in the script.

Reply:
yes